packages/meditrak-app/app/version/isBeta.jsx

```diff
@@ -7,4 +7,5 @@ import Config from 'react-native-config';
 
 export const isBeta = !!Config.BETA_BRANCH;
 export const betaBranch = Config.BETA_BRANCH;
-export const centralApiUrl = Config.CENTRAL_API_URL;
+export const centralApiUrl = 'http://10.0.2.2:8090/v2';
```

Comment:
I assume that this is just test code?

Reply:
Gah I thought I had unticked that one, will remove!